scripts/dataset/logger/index.js

```diff
@@ -0,0 +1,21 @@
+import winston from 'winston';
+
+const { combine, timestamp, printf, colorize } = winston.format;
+
+const alignedWithColorsAndTime = combine(
+  colorize(),
+  timestamp({ format: 'YYYY-MM-DD HH:mm:ss' }),
+  printf(({ level, message, timestamp }) => `${timestamp} ${level.padEnd(15)} ${message}`),
+);
+
+const consoleTransport = new winston.transports.Console();
```

Comment:
So no email notifications in case of publishing failures… This is not very satisfactory. Couldn't we use the main notifier setup?

Reply:
You mean the main logger setup. Yep I will do that.